src/css/profile/mobile/common/calendar.less

```diff
@@ -84,10 +84,10 @@
 	}
 	
 	&-prev-month-day {
-		opacity: 0.1;
+		opacity: unit(10%) / 100;
```

Comment:
Let's leave it as it was. The `unit` has applies in specific cases.

Reply:
eg. to covert `less` variables

Reply:
@TomaszLukawskiSam 
I use it in caledar.less, is it better to return the code to the previous one?

Reply:
I prefer previous, sorry.